README.md

```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add a description about the project to make your repo appear more professional.